ios/FluentUI/Button/Button.swift

```diff
@@ -188,9 +175,7 @@ open class Button: UIButton {
 
         if let image = image(for: .normal) {
             size.width += image.size.width
-            if #available(iOS 15.0, *) {
-                size.width += style.titleImagePadding
-            }
+            size.width += style.titleImagePadding
```

Comment:
Should we condense lines 177 + 178?

Reply:
Turns out if we just flip the logic below, we can get rid of it